public/keymaps/nafuda_default .json

```diff
@@ -0,0 +1,28 @@
+{
+  "keyboard": "naked60/rev1",
+  "keymap": "default_ad36bb3",
+  "commit": "687a24f2980e6e490ac8e43104f4176b033c9243",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+                          "TG(1)",
+"LT(3, KC_BSPC)",         "KC_UP",     "TG(2)",
+       "KC_LEFT",       "KC_DOWN",   "KC_RIGHT"
+    ],
+    [
+                          "TG(1)",
+        "KC_BTN1",      "KC_MS_U",   "KC_BTN2",
+        "KC_MS_L",      "KC_MS_D",    "KC_MS_R"
+    ],
+    [
+                     "LCTL(KC_W)",
+    "LCTL(LSFT(KC_T))", "KC_WH_U",     "TG(2)",
+  "LCTL(LSFT(KC_TAB))", "KC_WH_D", "LCTL(KC_TAB)"
+    ],
+    [
+                        "RGB_VAD",
+          "MO(3)",      "RGB_SAD",   "RGB_VAI",
+        "RGB_MOD",      "RGB_TOG",   "RGB_SAI"
+    ]
+  ]
+}
```

Comment:
And after you've done all of :point_up: , please remove the space in the filename.

```
git mv "public/keymaps/nafuda_default .json" public/keymaps/nafuda_default.json
```

Reply:
Thank you for pointing out.